### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by szepeviktor** [pods-framework/pods-export-code#1]
Could you explain what is the benefit of the exported PHP code?
This way I shouldn't create pods by hand??

``` php
function register_my_pods_config_3257390() {

    $pod = array (
        'name' => 'any',
        'label' => 'any',
        'type' => 'post_type',
        'storage' => 'meta',
        'show_in_menu' => '1',
        'label_singular' => 'any',
        'public' => '1',
        'show_ui' => '1',
        'supports_title' => '1',
        'supports_editor' => '1',
        'publicly_queryable' => '1',
        'exclude_from_search' => '0',
        'capability_type' => 'post',
        'capability_type_custom' => 'any',
        'capability_type_extra' => '1',
        'has_archive' => '0',
        'hierarchical' => '0',
        'rewrite' => '1',
        'rewrite_with_front' => '1',
        'rewrite_feeds' => '0',
        'rewrite_pages' => '1',
        'query_var' => '1',
        'can_export' => '1',
        'default_status' => 'draft',
        'supports_author' => '0',
        'supports_thumbnail' => '0',
        'supports_excerpt' => '0',
        'supports_trackbacks' => '0',
        'supports_custom_fields' => '0',
        'supports_comments' => '0',
        'supports_revisions' => '0',
        'supports_page_attributes' => '0',
        'supports_post_formats' => '0',
        'built_in_taxonomies_category' => '0',
        'built_in_taxonomies_link_category' => '0',
        'built_in_taxonomies_post_tag' => '0',
        'menu_position' => '0',
        'show_in_nav_menus' => '1',
        'show_in_admin_bar' => '1',
    );

    pods_register_type( $pod[ 'type' ], $pod[ 'name' ], $pod );

    $field = array (
        'name' => 'egy',
        'label' => 'egy',
        'type' => 'text',
        'weight' => 0,
        'text_allow_shortcode' => '0',
        'text_allow_html' => '0',
        'text_allowed_html_tags' => 'strong em a ul ol li b i',
        'text_max_length' => '255',
        'text_repeatable' => 0,
    );

    pods_register_field( $pod[ 'name' ], $field[ 'name' ], $field );

}
add_action( 'init', 'register_my_pods_config_3257390' );
```

**Comment by sc0ttkclark** [comment]
This is a beta plugin still in development. Code registered Pods and fields work for Pods 2.x but there are quarks we're working on for relationship (non-simple) fields and table-based Pods which may not get fixes until Pods 3.1. If you're using `define( 'PODS_TABLELESS_MODE', true );` then everything should work as intended.

Registering via code is great because it can be versioned with your theme or plugins and be more easily collaborated on with other developers on the same team.

**Comment by szepeviktor** [comment]
Thank you very much!
